RLBotFramework/utils/class_importer.py

```diff
@@ -1,75 +1,74 @@
 import importlib
 import inspect
-
 import os
+import sys
 
 from RLBotFramework.agents.base_agent import BaseAgent
-from RLBotFramework.utils.logging_utils import log_warn
 
 
-def is_extends_base_class(module, base_class):
-    super_classes = module.__bases__
-    for super_class in super_classes:
-        if super_class == base_class:
-            return True
-    return False
+class AgentLoadData:

```

Comment:
it looks like this applies to agents and extensions.
Should we come up with a name that applies to both?

Reply:
Sure, I'll rename it.